backend/distributor/serializers.py

```diff
@@ -52,13 +56,13 @@ class Meta:
 
 class HospitalSerializer(serializers.HyperlinkedModelSerializer):
     phone_numbers = HospitalPhoneNumberSerializer(many=True, read_only=True)
-    statistics = StatisticPhoneNumberSerializer(many=True, read_only=True)
     location = serializers.SerializerMethodField()
+    statistics = StatisticPhoneNumberSerializer(many=True, read_only=True)
     needs = HospitalNeedsSerializer(many=True, read_only=True)
 
     class Meta:
         model = Hospital
-        fields = ['id', 'name', 'code', 'address', 'location', 'locality_id', 'phone_numbers', 'statistics', 'needs']
+        fields = ('id', 'name', 'code', 'address', 'location', 'phone_numbers', 'needs', 'locality_id', 'statistics')
```

Comment:
Let's put locality_id after location.

Reply:
Is the location of the fields important?

Reply:
:) frontend guys then will ask

Reply:
Done